packages/docusaurus-theme-classic/src/theme/NavbarItem/DocNavbarItem.tsx

```diff
@@ -12,17 +12,10 @@ import clsx from 'clsx';
 import type {Props} from '@theme/NavbarItem/DocNavbarItem';
 import {useDocsPreferredVersion} from '@docusaurus/theme-common';
 import {uniq} from '@docusaurus/utils-common';
-import type {
-  GlobalDataVersion,
-  GlobalDataDoc,
-} from '@docusaurus/plugin-content-docs-types';
+import type {GlobalDataVersion} from '@docusaurus/plugin-content-docs-types';
 
 function getDocInVersions(versions: GlobalDataVersion[], docId: string) {
-  // vanilla-js flatten, TODO replace soon by ES flat() / flatMap()
-  const allDocs: GlobalDataDoc[] = [].concat(
```

Comment:
not 100% sure but I think we need to keep it for older node/browsers

Reply:
I think this is taken care of by Babel, as long as the target is set to `ES5` or `ES6`?

Reply:
yes babel probably does something with browserslist

I think I avoided flatMap in the past due to having to support node10